packages/manager/modules/pci/src/projects/project/instances/instance/shelve/shelve.controller.js

```diff
@@ -0,0 +1,77 @@
+import get from 'lodash/get';
+import Instance from '../../../../../components/project/instance/instance.class';
+
+export default class PciInstanceShelveController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    OvhApiCloudProjectInstance,
```

Comment:
Not used.

Reply:
In the previous version yes.
I made a cleanup.